migrate.js

```diff
@@ -49,6 +50,8 @@ const wrapCommand = fn => async options => {
 
   const confirm = async (msg, def) => {
     if (force) {
+      return true
```

Comment:
probably should remove the `def` parameter here, or certainly not override it.

I think it is the caller's responsibility to know what the default response should be in a given confirmation context.  There should probably not be a default value defined inside of `confirm`, in other words, `def` should be a required parameter, never undefined.

Reply:
Why? You must have the default value to allow a default options. Also you don't always want to have a default response, so it makes sense to have it undefined.

Reply:
The knowledge about what the default should be lies outside of the confirm function.  The confirm function is used in several contexts where "default" has a completely different sense, only known to the caller.  This function violates the single responsibility function, and the result will be code that is less clean, robust and flexible.

Reply:
The caller will know what the default is and will not invoke `confirm` unless a confirmation is truly needed.

Reply:
> This function violates the single responsibility function, and thus the code is less clean, robust and flexible.

For example, overriding `def` when `force` begs the question of in what context does `force` even make sense?  Same with `disableConfs` and using `true` as the default.  How can this function know that `true` necessarily makes sense in every possible context?